ecs_fargate/README.md

```diff
@@ -595,7 +595,7 @@ For more information on CloudFormation templating and syntax, see the [AWS Cloud
 
 ### Trace collection
 
-1. Follow the [instructions above](#installation) to add the Datadog Agent container to your task definition with the additional environment variable `DD_APM_ENABLED` set to `true` and set up a host port that uses **8126** with **tcp** protocol under port mappings. Set the `DD_SITE` variable to {{< region-param key="dd_site" code="true" >}}. It defaults to `datadoghq.com` if you don't set it.
+1. Follow the [instructions above](#installation) to add the Datadog Agent container to your task definition with the additional environment variable `DD_APM_ENABLED` set to `true` and set up a container port that uses **8126** with **tcp** protocol under port mappings. Set the `DD_SITE` variable to {{< region-param key="dd_site" code="true" >}}. It defaults to `datadoghq.com` if you don't set it.
```

Comment:
I’m not sure it’s correct to replace `host port` by `container port`.
I think it’s really a `host port` that the agent needs to configure in order to be reachable by other containers running on the same host.

In [this doc](https://docs.datadoghq.com/containers/amazon_ecs/apm/?tab=ec2metadataendpoint), it’s a `hostPort` that is configured.